addons/website/views/snippets/s_intro_pill.xml

```diff
@@ -0,0 +1,55 @@
+<odoo>
+
+<template id="s_intro_pill" name="Intro Pill">
+    <section class="s_intro_pill pt40 pb40">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item oe_img_bg o_grid_item_image g-col-lg-5 g-height-9 col-lg-5 order-lg-0" style="order: 1; z-index: 1; grid-area: 1 / 1 / 10 / 6; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img-fluid mx-auto" data-shape="web_editor/geometric_round/geo_round_pill" data-file-name="s_intro_pill_default_image.jpg"
+                    data-original-mimetype="image/jpeg" src="/web_editor/image_shape/website.s_intro_pill_default_image/web_editor/geometric_round/geo_round_pill.svg"/>
+                </div>
+                <div class="o_grid_item g-height-11 g-col-lg-8 col-lg-8 justify-content-center order-lg-0" data-name="Box" style="order: 2; z-index: 3; grid-area: 1 / 3 / 12 / 11; text-align: center;">
+                    <h1 class="display-3">Experience the<br/>best quality services</h1>
+                    <p>
+                        <br/>
+                    </p>
+                    <p>
+                        <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary">Get Started</a>
+                    </p>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-5 g-height-10 d-none d-lg-block col-lg-5" style="z-index: 2; grid-area: 2 / 8 / 12 / 13; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img-fluid mx-auto" data-shape="web_editor/geometric_round/geo_round_pill" data-file-name="s_intro_pill_default_image_2.jpg" data-original-mimetype="image/jpeg" src="/web_editor/image_shape/website.s_intro_pill_default_image_2/web_editor/geometric_round/geo_round_pill.svg"/>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="s_intro_pill_options" inherit_id="website.snippet_options">
+    <!--  Box Vertical Alignment -->
+    <xpath expr="//div[@data-js='WebsiteAnimate']" position="after">
+        <div data-selector=".s_intro_pill .o_grid_item" data-exclude=".o_grid_item_image">
+            <we-button-group string="Vert. Alignment" title="Vertical Alignment">
+                <we-button title="Align Top"
+                        data-select-class=""
+                        data-img="/website/static/src/img/snippets_options/align_solo_top.svg"/>
+                <we-button title="Align Middle"
+                        data-select-class="justify-content-center"
+                        data-img="/website/static/src/img/snippets_options/align_solo_middle.svg"/>
+                <we-button title="Align Bottom"
+                        data-select-class="justify-content-end"
+                        data-img="/website/static/src/img/snippets_options/align_solo_bottom.svg"/>
+            </we-button-group>
+        </div>
```

Comment:
If you toggle the block to Column mode, this option remains displayed while it has no more effect.
You can disable it by implementing `_computeWidgetVisibility` as described in that comment <https://github.com/odoo/odoo/pull/175746#discussion_r1715646403>.

Reply:
I changed a bit the implementation. We use now grid padding to center the element so we don't have to add extra scss, options, js etc...

The visual result is basically the same except on mobile where I changed the order of the elements so we can have the title in the viewport

|  | Before | After |
|--------|--------|--------|
| Desktop | <img width="1710" alt="Screenshot 2024-08-30 at 12 34 13" src="https://github.com/user-attachments/assets/a7e9ca7e-d5b5-417a-acc2-b9925aff4f8c"> | <img width="1708" alt="Screenshot 2024-08-30 at 12 32 56" src="https://github.com/user-attachments/assets/521546cd-e537-4e43-99d5-c770f562ceb6"> |
| Mobile | <img width="398" alt="Screenshot 2024-08-30 at 12 34 35" src="https://github.com/user-attachments/assets/bf2b18f4-ddb9-4b6a-9b6d-bfac5d64542f"> | <img width="398" alt="Screenshot 2024-08-30 at 12 33 21" src="https://github.com/user-attachments/assets/5933c30c-5a1f-49ae-95b9-f4a11fe5e5f1"> |